js/plates-view/render-cross-section.ts

```diff
@@ -679,6 +679,11 @@ class CrossSectionRenderer {
     ctx.ellipse(scaleX(avgLocation.x), scaleY(avgLocation.y), 0.45 * scaledWidth, 0.4 * scaledHeight,
       2 * Math.PI, 0, 2 * Math.PI);
 
+    // uncomment these lines to visualize/debug the hit-testing shape
+    // ctx.lineWidth = 1;
+    // ctx.strokeStyle = "black";
+    // ctx.stroke();
+
```

Comment:
I think I've accidentally removed this code before (it was yours and useful for debugging), so I'm re-adding it here.